yarn-project/end-to-end/src/shared/cross_chain_test_harness.ts

```diff
@@ -346,7 +362,7 @@ export class CrossChainTestHarness {
     expect(balance).toBe(expectedBalance);
   }
 
-  async checkEntryIsNotInOutbox(withdrawAmount: bigint, callerOnL1: EthAddress = EthAddress.ZERO): Promise<Fr> {
+  getL2ToL1MessageLeaf(withdrawAmount: bigint, callerOnL1: EthAddress = EthAddress.ZERO): Fr {
     this.logger('Ensure that the entry is not in outbox yet');
```

Comment:
This is not ensured anymore. Before it was essentially checking that it was not there. So think we should remove the log here as we are not doing that anymore.